WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewHolder.kt

```diff
@@ -157,4 +159,14 @@ sealed class BloggingRemindersViewHolder<T : ViewBinding>(protected val binding:
             uiHelpers.setTextOrHide(message, item.message)
         }
     }
+
+    class TimeViewHolder(parentView: ViewGroup, private val uiHelpers: UiHelpers) :
+            BloggingRemindersViewHolder<BloggingRemindersTimeBinding>(
+                    parentView.viewBinding(BloggingRemindersTimeBinding::inflate)
+            ) {
+        fun onBind(item: TimeItem) = with(binding) {
+            uiHelpers.setTextOrHide(timeButton, item.time)
+            timeButton.setOnClickListener { item.onClick.click() }
```

Comment:
We should probably set the click listener on the whole view, instead of just the `TextView`. Otherwise, the touch target gets really small and easy to miss.

Reply:
I was wondering the same, should the padding needs to added here around time or just make the whole thing touch target.  I'll change to the whole view for now.

Reply:
Update touch target to the list item

Reply:
Making the whole view the target seems the right thing to do 👍 I'd just make sure to add the selectable background to the view, so we can see ripple animation when we touch it.

Reply:
added selectableItemBackground.  Thanks